includes/model/class-openedx-woocommerce-plugin-log.php

```diff
@@ -87,6 +87,9 @@ public function check_api_response( $response ) {
 			case 'not_api':
 				return 'Enrollment saved locally, action does not require an API call';
 
+			case 'no_process':
```

Comment:
If we are going to create this case, do we need the default case? In what scenario does the plugin return 'API did not provide a response'?
Can we use a return for the case 'no_process' more descriptive? In the other case, we return a string, but in that specific case, we return a response[1]. Is it not better to return the message you defined, 'The enrollment process has been skipped.', here?